### PR TITLE
ACMS-1239: As a developer, I want ACMS low-code to be pre-configured …

### PR DESCRIPTION
**Pull request by chandan-singh7929**
…with recommended Site Studio configuration management approach.

**Motivation**

Fixes #ACMS-1239[https://backlog.acquia.com/browse/ACMS-1239]

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good.

File: modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# About
+
+Contains configuration for config ignore & config split specific to site studio module, that allow better configuration
+management between core config and site studio.
+
+## Usages
+In order to utilise this module's functionality, user needs to add scaffolding to their project.
+they can run command `composer config --json extra.drupal-scaffold.allowed-packages ["acquia/acquia_cms_site_studio"] --merge && composer instal`
```
Comment (vishalkhode1) [inline modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md:8]:
```suggestion
they can run command 
```composer config --json extra.drupal-scaffold.allowed-packages ["acquia/acquia_cms_site_studio"] --merge && composer install```
```
The install

File: modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+will do the scaffolding, and then enabling this module will create a config split named _sitestudio_
+along with a config ignore and default Site Studio full export settings.
+- Manually do scaffolding
+- Enable this module.
+- Run drush command `drush cex`.
+
+### Scaffolding
+This module provides composer scaffolding which updates default.settings.php to include _site_studio_sync_ directory path.
+`$settings['site_studio_sync'] = '../config/sitestudio';`. It also provides post commands for `drush cex` & `drush cim`,
```
Comment (vishalkhode1) [inline modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md:17]:
```suggestion
```$settings['site_studio_sync'] = '../config/sitestudio';```. It also provides post commands for `drush cex` & `drush cim`,
```

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

File: modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md (hunk 2 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md]: Done.

File: modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md (hunk 1 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md]: Done

**Comment by chandan-singh7929** [comment]
LGTM

**Pull request merged**